lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -421,6 +419,20 @@ newDBLayer trace fp timeInterpreter = do
         , atomically = runQuery
         })
     where
+        listRetiredPools_ epochNo = do
+            let query = T.unwords
+                    [ "SELECT * FROM "
+                    , databaseViewName activePoolRetirements
+                    , "WHERE retirement_epoch <="
+                    , T.toText epochNo
```

Comment:
> Isn't there a `?` parameter syntax that we should be using here.

I don't know, is there?

Reply:
Yes we must maintain a good habit of always using `?` value placeholders. Even though in this particular instance it is safe.

Injection is the number 1 OWASP risk.

Also I am not fond of using views when normal queries would work just as well.

Reply:
> Yes we must maintain a good habit of always using ? value placeholders. Even though in this particular instance it is safe.

Fair enough.

Since this PR no longer touches this area of the code, I will create a separate PR to add `?` value placeholders.